core/src/test/scala/integration/kafka/api/SaslMultiMechanismConsumerTest.scala

```diff
@@ -35,7 +35,7 @@ class SaslMultiMechanismConsumerTest extends BaseConsumerTest with SaslTestHarne
     val plainSaslProducer = producers(0)
     val plainSaslConsumer = consumers(0)
 
-    val gssapiSaslProperties = kafkaSaslProperties("GSSAPI", kafkaServerSaslMechanisms)
+    val gssapiSaslProperties = kafkaSaslProperties("GSSAPI")
```

Comment:
OK, because we don't want to set the sasl enabled mechanisms config for the clients.